actionview/lib/action_view/helpers/csp_helper.rb

```diff
@@ -14,9 +14,11 @@ module CspHelper
       # This is used by the Rails UJS helper to create dynamically
       # loaded inline <script> elements.
       #
-      def csp_meta_tag
+      def csp_meta_tag(**options)
         if content_security_policy?
-          tag("meta", name: "csp-nonce", content: content_security_policy_nonce)
+          options[:name] = "csp-nonce"
+          options[:content] = content_security_policy_nonce
+          tag("meta", options)
```

Comment:
Could switch to tag.meta while we’re here.

Reply:
What about keeping the line the same but unsplatting the options at the end of the tag call?

Reply:
In `tag("meta")` and `tag.meta`, the generated tags are slightly different. 

```ruby 
helper.tag("meta", name: "csp-nonce")
#=> "<meta name=\"csp-nonce\" />"
helper.tag.meta(name: "csp-nonce")
#=> "<meta name=\"csp-nonce\"> 
``` 

Of course, since meta tag is a void element, it is not necessary to have a close tag. However, the tag generated by `csrf_meta_tags` have a close tag, and I think that it is better to behave similarly so that the closing tag is generated. 

> What about keeping the line the same but unsplatting the options at the end of the tag call? 

:+1: I fixed.

Reply:
> What about keeping the line the same but unsplatting the options at the end of the tag call?

It would cause a warning https://travis-ci.org/rails/rails/jobs/493601600#L1254-L1256.

Reply:
I screwed up! Thanks again!